### DISCUSSION TITLE
Compilation error: soc/rtc_wdt.h: No such file or directory

### DISCUSSION THREAD

**Discussion by martin230896** [espressif/arduino-esp32#9800] [Q&A] [answered]
Hello, I am facing a problem after I updated my ESP32 board to version 3.0.0 using Arduino IDE 2.3.2 The following error appears: Compilation error: soc/rtc_wdt.h: No such file or directory

**Comment by SXlaowang** [comment]
#include <rtc_wdt.h>

**Comment by nathmo** [comment, marked as answer]
as @SXlaowang  said but using more words : 
replace 
`#include <soc/rtc_wdt.h>`
by
`#include <rtc_wdt.h>`

and feel free to mark as closed if it works :)

**Comment by Renato1943** [comment]
#include <rtc_wdt.h> Perfeito, o meu código estava dando esse erro e eu já estava apanhando há quase um mês, retirei do código SOC e deu tudo certo.
Obrigado amigo, Valeu a dica
